shared-libs/transitions/src/transitions/registration.js

```diff
@@ -505,6 +525,14 @@ const addPatient = (options) => {
           patient.date_of_birth = doc.birth_date;
         }
 
+        if (patientPhoneField && doc.fields[patientPhoneField]) {
+          if (!phoneNumberParser.validate(config.getAll(), (doc.fields[patientPhoneField]))) {
```

Comment:
To make this whole block more readable, please store `doc.fields[patientPhoneField]` in a variable. 
```suggestion
const patientPhone = doc.fields[patientPhoneField];
if (!phoneNumberParser.validate(config.getAll(), patientPhone)) {
....
  return;
}
patient.phone = patientPhone;
```

Reply:
done.